airbyte_cdk/utils/docker.py

```
@@ -17,6 +17,8 @@
 import requests
 
 from airbyte_cdk.models.connector_metadata import ConnectorLanguage, MetadataFile
+from airbyte_cdk.sources.declarative import spec
```

Flake8 (7.2.0)

[error] 'airbyte_cdk.sources.declarative.spec' imported but unused

(F401)